packages/core/src/errors/useroperation.ts

```diff
@@ -13,3 +13,10 @@ export class InvalidUserOperationError extends BaseError {
     );
   }
 }
+
+export class MultisigMissingSignatureError extends BaseError {
```

Comment:
this is a super nit pick... but can we move this error into accounts / msca? it's not really relevant anywhere but within aa-accounts and specifically for MSCA

Reply:
yeah I thought it was odd that it was in aa-core at all. Happy to make that move, but would it make more sense to define the error locally in aa-accounts?